cylc/flow/config.py

```diff
@@ -1531,24 +1530,37 @@ def configure_workflow_state_polling_tasks(self):
                 continue
             rtc = tdef.rtconfig
             comstr = (
-                "cylc workflow-state"
-                f" --task={tdef.workflow_polling_cfg['task']}"
-                " --point=$CYLC_TASK_CYCLE_POINT"
+                "cylc workflow-state "
```

Comment:
Am I correct in thinking that this loop is only called on the old graph-syntax polling system?

I tried

```
[scheduler]
    allow implicit tasks = True
    cycle point format = %Y
[scheduling]
    initial cycle point = 2010
   [[graph]]
        P1Y = """
            foo<WORKFLOW::TASK:fail> => bar
        """
```

expecting a deprecation warning, but didn't get one.

Reply:
You are correct, but we only print deprecation warnings for config items that get automatically upgraded. In this case you don't have a `[[[suite state polling]]]` section that triggers the warning.

I've added an additional deprecation warning at validation time, via the config module:
```
WARNING - Workflow state polling tasks are deprecated, please convert to workflow_state xtriggers:
     * oof
     * woof
```

Reply:
I'm sure it'd not be the first time we've had to write custom config warnings outside the parsec deprecation framework....